modules/gdnative/gdnative/string.cpp

```diff
@@ -125,6 +122,35 @@ void GDAPI godot_string_new_with_wide_chars_and_len(godot_string *r_dest, const
 	}
 }
 
+const char GDAPI *godot_string_to_latin1_chars(const godot_string *p_self) {
+	String *self = (String *)p_self;
+	return self->ascii(true).get_data();
+}
+
+const char GDAPI *godot_string_to_utf8_chars(const godot_string *p_self) {
```

Comment:
For my understanding: I thought the new gdnative api was exposing the builtins methods through discovery functions, why is it not the case for those ?

Reply:
Those are not in the Variant API, because Variant doesn't know what `const char *` is. That's why this have String to C string conversion functions. The other direction was added already in another PR.

Reply:
👍